src/jit/lir.cpp

```diff
@@ -385,6 +434,7 @@ bool LIR::Range::IsValid() const
 //
 bool LIR::Range::IsEmpty() const
 {
+    // TODO: if "FirstNode() == nullptr", also assert LastNode()==nullptr?
```

Comment:
Yes, the assert would be good--this property is expected to be correct by construction.